drivers/hmis/app/jobs/hmis/migrate_assessments_job.rb

```diff
@@ -62,18 +64,22 @@ def perform(data_source_id:, project_ids: nil, clobber: false, delete_dangling_r
           each(&:really_destroy!)
       end
 
+      total = full_enrollment_scope.count
+      Rails.logger.info "#{total} Enrollments to process"
+
       # Note: joining with project drops WIP enrollments. That should be fine since they won't have assessment records yet.
```

Comment:
Maybe would be good to move this comment down to line 110 where the join is now happening?